app/api/v1/routes/games.py

```
@@ -0,0 +1,24 @@
+from app.infrastructure.database.session import get_database_session
+from app.logger import get_logger
+from app.schemas.game import GameCreate
+from app.infrastructure.database.repositories.user_repository import UserRepository
+
+
+logger = get_logger(__name__)
+
+game_router = APIRouter(prefix="/game", tags=["game"])
+
+@game_router.post("/initiate/{creator_id}", summary="Initiate a new game", status_code=201)
+async def initiate_new_game_endpoint(creator_id: int, game_data: GameCreate, request: Request, session: AsyncSession = Depends(get_database_session)): #TODO: this should get game data from request, just for test
```

Ruff (0.15.17)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)